server/@types/ui/index.d.ts

```diff
@@ -401,16 +401,12 @@ type ContingencyPlanQuestion = {
 
 export type ContingencyPlanQuestionsRecord = Record<ContingencyPlanQuestionId, ContingencyPlanQuestion>
 
-export interface BedSearchParametersUi {
```

Comment:
Are we confident that this code is not being used at present? Generally I've gone for duplicating code for the v2 functionality, to minimise risk of breaking things. But if we're confident it's not being used then it would be better to replace it like you have.

Reply:
The only place it is being used is this search page. This page is quite standalone whereas the booking journey that you've been working on is a bit more intermingled with in-use code so I think we've both chosen the right approach despite them being distinct.